### PR TITLE
Adding modifications to the Readme file

### PR DESCRIPTION
**Pull request by cmarin21**
Created some modifications in the Readme file. Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,16 @@ Test Repository
 ===============
 
 Playing around with Git and GitHub.
+
+This is my modification, Claudia.
+
+This week I am in Suse Nurnberg.
```
Comment (tomschr) [inline README.md]: Could you please replace Suse -> SUSE?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,16 @@ Test Repository
...
+More modifications into Readme.md
+
+--
+
+Cities
```
Comment (tomschr) [inline README.md]: Could you make a section of that? Use "-" underlines.

Comment (cmarin21) [reply README.md]: Modified according to your suggestions.

**Comment by tomschr** [comment]
Test

**Comment by cmarin21** [comment]
This is just a comment from my side, related to pull request.

**Pull request merged**